1.process_data/README.md

```diff
@@ -0,0 +1,17 @@
+# Merge single cells and normalize using pycytominer
+
+In this module, we perform two preprocessing steps on the SQLite files using [pycytominer](https://github.com/cytomining/pycytominer/tree/main):
+
+1. Merge and annotate single cells from the SQLite file using the [pycytominer SingleCell class](https://github.com/cytomining/pycytominer/blob/main/pycytominer/cyto_utils/cells.py)
+2. [Normalize](https://github.com/cytomining/pycytominer/blob/main/pycytominer/normalize.py) the single cells using the negative controls (DMSO treatment) as reference for the standard scalar method.
```

Comment:
```suggestion
2. [Normalize](https://github.com/cytomining/pycytominer/blob/main/pycytominer/normalize.py) the single cells using the negative controls (e.g., DMSO treatment for compound plates) as reference for the standard scalar method.
```

Reply:
b/c DMSO is not neg control for all plates

Reply:
I have included what the negative controls are for the other two treatment types. Thanks for pointing that out!